packages/web-extension/src/cip30/initializeContentScript.ts

```diff
@@ -12,6 +12,7 @@ export const initializeContentScript = (
   { injectedScriptSrc, walletName }: InitializeContentScriptProps,
   dependencies: MessengerDependencies
 ) => {
+  // TODO: How to pass both with the channelName as apis {[channelName]: ApiObject}
```

Comment:
This function should receive a record of APIs like `Record<ApiName, AnyApi>`

Reply:
Fixed on f401a49